src/main/java/soconnect/ui/CommandBox.java

```diff
@@ -90,21 +95,13 @@ private void setAutocompleteListener() {
             }
         });
 
-        commandTextField.setOnKeyReleased(new EventHandler<KeyEvent>() {
+        commandTextField.textProperty().addListener(new ChangeListener<String>() {
             @Override
-            public void handle(KeyEvent event) {
-                switch (event.getCode()) {
-                case UP:
-                case DOWN:
-                case LEFT:
-                case RIGHT:
-                case SPACE:
-                    break;
-                default:
-                    autocompleteAction();
-                }
```

Comment:
What's the consequence of erasing the UP DOWN, etc part?

Reply:
Nothing will happen. Initially, I made the Key listener not generate autocomplete when the UP DOWN keys are pressed because I wanted to update the text in the command box when the user chooses the autocomplete entries using the arrow key. But since I can't figure out a way to make this behaviour happen, so I decided to remove it.